boards/fobe/quill_nrf52840_mesh/doc/index.rst

```diff
@@ -0,0 +1,162 @@
+.. zephyr:board:: quill_nrf52840_mesh
+
+Overview
+********
+
+The FoBE Quill nRF52840 Mesh is a development kit featuring the nRF52840 SoC and an integrated SX1262 LoRa® transceiver.
+
+For more details see the `FoBE Quill nRF52840 Mesh`_ documentation page.
+
+.. figure:: img/quill_nrf52840_mesh.webp
+   :align: center
+   :alt: Quill nRF52840 Mesh
+
+   Quill nRF52840 Mesh
```

Comment:
not needed since zephyr:board:: directive takes care of including the image on the page

```suggestion
```